### PR TITLE
chore(docs): add Chinese README

### LINKED ISSUE #194: [Feat][Doc] Add README and document for zh_cn version
as title

### PR DESCRIPTION
**Pull request by yecol**
### Reason for this PR

To fix #194 

### What changes are included in this PR?

Added Chinese version readme.

### Are these changes tested?

Yes.

### Are there any user-facing changes?

No.

### CODE REVIEW

File: README-zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+width="650" alt="vertex physical table" />
+
+> [!NOTE]
+> 为了有效利用诸如 Parquet 之类 payload 文件格式的过滤下推功能，内部顶点ID作为一列存储在 payload 文件中。由于内部顶点ID是连续的，payload 文件格式可以对内部顶点ID列使用增量编码，这不会给存储带来太多的开销。
+
+### GraphAr 的边
+
+#### 边的逻辑表
+
+为了维护一种类型的边（具有相同的源标签、边标签和目标标签），会建立一个逻辑边表。为了支持从图存储文件中快速创建图，逻辑边表可以以类似于CSR/CSC（了解更多关于CSR/CSC）的方式维护拓扑信息，即边按照源或目标的内部顶点ID排序。通过这种方式，需要一个偏移表来存储每个顶点的边的起始偏移量，并且具有相同源/目标的边将连续存储在逻辑表中。
```
Comment (acezen) [inline README-zh-cn.md:84]: The link of `CSR/CSC` is missing

**Review by acezen — APPROVED** [review]
LGTM, @Thespica , could you take a look to this PR?

File: README-zh-cn.md (hunk 1 above)
Comment (yecol) [reply README-zh-cn.md]: fixed.

File: README-zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+</p>
+
+[![GraphAr
+CI](https://github.com/apache/incubator-graphar/actions/workflows/ci.yml/badge.svg)](https://github.com/apache/incubator-graphar/actions)
+[![Docs
+CI](https://github.com/apache/incubator-graphar/actions/workflows/docs.yml/badge.svg)](https://github.com/apache/incubator-graphar/actions)
+[![GraphAr
+Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://graphar.apache.org/docs/)
+[![Good First
+Issue](https://img.shields.io/github/labels/apache/incubator-graphar/Good%20First%20Issue?color=green&label=Contribute)](https://github.com/apache/incubator-graphar/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```
Comment (Thespica) [inline README-zh-cn.md:15]: Maybe we can add an icon to indicate that there are other version languages of README? Same as English README.

File: README-zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+为了维护一种类型的边（具有相同的源标签、边标签和目标标签），会建立一个逻辑边表。为了支持从图存储文件中快速创建图，逻辑边表可以以类似于 [CSR/CSC](https://en.wikipedia.org/wiki/Sparse_matrix) 的方式维护拓扑信息，即边按照源或目标的内部顶点ID排序。通过这种方式，需要一个偏移表来存储每个顶点的边的起始偏移量，并且具有相同源/目标的边将连续存储在逻辑表中。
+
+以 person knows person 边的逻辑表为例，逻辑边表看起来如下所示：
+
+<img src="docs/images/edge_logical_table.png" class="align-center"
+width="650" alt="edge logical table" />
+
+#### 边的物理表
+与顶点表相同，逻辑边表也被分割为一些子逻辑表，每个子逻辑表包含源（或目标）顶点在相同顶点块中的边。根据分区策略和边的顺序，边可以按照以下四种类型之一存储在GraphAr中：
```
Comment (Thespica) [inline README-zh-cn.md:92]: Suggested by [this guide](https://github.com/sparanoid/chinese-copywriting-guidelines#%E4%B8%AD%E8%8B%B1%E6%96%87%E4%B9%8B%E9%96%93%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC), we can add space between Chinese and English for better readability(not only here).

**Review by Thespica — CHANGES_REQUESTED** [review]

**Comment by yecol** [comment]
Thanks @Thespica !
I made a revision.

**Review by Thespica — APPROVED** [review]
Thank you @yecol !

LGTM~

**Pull request merged**